Dockerfile

```diff
@@ -28,13 +28,15 @@ WORKDIR $CONFLUENCE_HOME
 
 COPY entrypoint.sh /entrypoint.sh
 
+# Download required Confluence version
 RUN [ -n "${CONFLUENCE_VERSION}" ] || export CONFLUENCE_VERSION=$(curl -s https://marketplace.atlassian.com/rest/2/applications/confluence/versions/latest | jq -r '.version') \
-    && echo "${CONFLUENCE_VERSION}" \
     && export DOWNLOAD_URL="http://www.atlassian.com/software/confluence/downloads/binary/atlassian-confluence-${CONFLUENCE_VERSION}.tar.gz" \
-    && echo "${DOWNLOAD_URL}" \
     && mkdir -p                          ${CONFLUENCE_INSTALL_DIR} \
-    && curl -L                           ${DOWNLOAD_URL} | tar -xz --strip-components=1 -C "$CONFLUENCE_INSTALL_DIR" \
-    && sed -i -e 's/-Xms\([0-9]\+[kmg]\) -Xmx\([0-9]\+[kmg]\)/-Xms\${JVM_MINIMUM_MEMORY:=\1} -Xmx\${JVM_MAXIMUM_MEMORY:=\2} \${JVM_SUPPORT_RECOMMENDED_ARGS} -Dconfluence.home=\${CONFLUENCE_HOME} -Dsynchrony.proxy.healthcheck.disabled=true/g' ${CONFLUENCE_INSTALL_DIR}/bin/setenv.sh
+    && curl -L                           ${DOWNLOAD_URL} | tar -xz --strip-components=1 -C "$CONFLUENCE_INSTALL_DIR"
+
+# Perform settings modifications
```

Comment:
No need to download again Confluence again because I changed the settings

Reply:
I like this.
This is a clever way to use docker layers!